test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -53,11 +53,11 @@ json_spirit::mValue BlockchainTestSuite::doTests(json_spirit::mValue const& _inp
 			continue;
 
 		BOOST_REQUIRE_MESSAGE(inputTest.count("genesisBlockHeader"),
-			"\"genesisBlockHeader\" field is not found. filename: " + TestOutputHelper::get().testFileName() +
+			"\"genesisBlockHeader\" field is not found. filename: " + TestOutputHelper::get().testFileName().string() +
```

Comment:
Why suddenly we have `.string()` everywhere?

Reply:
changed to boost path